source/administration/version-archive.rst

```diff
@@ -74,6 +77,9 @@ Mattermost Enterprise Edition v2.1.0 - `View Changelog <https://docs.mattermost.
 Mattermost Team Edition Server Archive
 ---------------------------------------
 
+Mattermost Team Edition v4.9.0 - `View Changelog <https://docs.mattermost.com/administration/changelog.html#release-v4-8>`_ - `Download <https://releases.mattermost.com/4.9.0/mattermost-team-4.9.0-linux-amd64.tar.gz>`_
```

Comment:
I notice the changelog is still referring to v4.8 - is this going to be updated once the v4.9 changelog is merged?